projects/layout/components/block-status/block-status.module.ts

```diff
@@ -0,0 +1,12 @@
+import {CommonModule} from '@angular/common';
+import {NgModule} from '@angular/core';
+import {TuiItemModule} from '@taiga-ui/cdk';
+
+import {TuiBlockStatusComponent} from './block-status.component';
+
+@NgModule({
+    imports: [CommonModule, TuiItemModule],
```

Comment:
`TuiItemModule` is it used?

Reply:
see [this comment](https://github.com/Tinkoff/taiga-ui/pull/4111#discussion_r1168767131)
we can create another directive, if `tuiItem` is not clear 🤔